src/incubator/TextField/index.tsx

```diff
@@ -84,15 +84,15 @@ export type TextFieldProps = MarginModifiers &
     /**
      * Internal style for the field container
      */
-    fieldStyle?: ViewStyle | ((context: FieldContextType) => ViewStyle);
+    fieldStyle?: ViewStyle | ((context: FieldContextType, props: {preset: TextFieldProps['preset']}) => ViewStyle);
```

Comment:
Why should we pass the preset to the user? we're getting it from the user and we don't change it inside the component..